src/Buffer.mo

```diff
@@ -1857,6 +2598,22 @@ module {
   /// elements with the same index. If one buffer is longer than the other, the
   /// remaining elements from the longer buffer are not included.
   ///
+  /// Example:
+  /// ```motoko include=initialize
+  ///
+  /// let a = Buffer.Buffer<Nat>(2);
+  /// a.add(1);
+  /// a.add(2);
+  /// a.add(3);
+  ///
+  /// let b = Buffer.Buffer<Nat>(2);
+  /// b.add(4);
+  /// b.add(5);
+  /// b.add(6);
+  ///
+  /// Buffer.zip(a, b);
```

Comment:
Should display this buffer using `toText` or `toArray` like in the other examples